README.md

```diff
@@ -111,6 +102,18 @@ Use `it.only(() => {})` and `describe.only(() => {})` if you want to run a speci
 For end to end testing we use the headless chrome. So it is mandatory to install the chrome browser for integration tests.
 Since Chrome 59 the headless mode is integrated. So please install a Chome 59 or higher to execute the end to end tests.
 
+#### Releasing
+Run `VERSION=<VERSION_YOU_WANT_TO_RELEASE> make release`
+e.g `VERSION=1.0.2 make release`.
+This checks if you set the environment variable, reinstall all node_modules,
+builds a production release runs `make lint`, `make test` and `make test-e2e`,
+bumps the version locally via lerna and publish the version as a new npm tag.
+After that you should carefully revisit your local changes, commit then and
+open a pull request on Github. When travis runs through merge it an tag a new
+release on Github.
+
+After that trigger jenkins with the new version.
```

Comment:
Could Jenkins do everything, including `make release`?

Reply:
In general yeah. We need to give him access to master branch and should find some way to only commit stuff we expect to change

Reply:
Master is generally protected, I don't know how to change that especially only for jenkins, we should not be able to push into master.